openshift/run-2e2-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+here=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+F8A_API_URL=$(oc get route bayesian-api | awk '{print$2}' | grep -w "${OC_USERNAME}")
+F8A_JOB_API_URL=$(oc get route bayesian-jobs | awk '{print$2}' | grep -w "${OC_USERNAME}")
+F8A_ANITYA_API_URL=$(oc get route bayesian-api | awk '{print$2}' | grep -w "${OC_USERNAME}")
+
+#install python dependencies
+pip install --user -r $here/../integration-tests/requirements.txt
+
+#check for configuration
+if [ "${F8A_API_URL}" == "Not set" ]; then
```

Comment:
Where do we set these variables to "Not set"?

Reply:
https://github.com/fabric8-analytics/fabric8-analytics-common/pull/394/commits/ba5a4fcd29f9275978c22d3b78a8a95fb64a650a